test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -7,6 +7,7 @@ Feature: Bind values from a secret referred in backing service resource
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
 
+    @disabled
```

Comment:
why do we want to disable this scenario now?

Reply:
Because #838 is not fixed yet so this scenario will fail.

Reply:
@pratikjagrut are you sure this scenario will fail ? since it did not @disabled tag, This tests runs for each CI test and we haven't seen any failure.

Reply:
It's not failing currently in CI but the test is modified little bit now to do regression check so until that bug is this regression will fail.

Reply:
got it